### ISSUE TITLE
GUI does not appear

### ISSUE THREAD

**Issue by Sciccorz** [0mlml/chesshook#18]
On both windows and mac, mian window does not appear, however the console and bot config windows are able to be displayed with their respective keybinds. Tried uninstalling and reinstalling script, updating browser, and restarting system

**Comment by 0mlml** [comment]
Uninstalling/reinstalling the script won't do anything as the script stores its state in localstorage. Could you try clearing the site data or typing `localStorage.clear()` into the developer console and refreshing to see if it works? While you're there, do you mind checking for any errors that the script might be causing?

**Comment by Sciccorz** [comment]
<img width="543" alt="image" src="https://github.com/user-attachments/assets/84e42608-53ae-4696-bab0-f5caa41bb353">
well these are what i see. It showed up once, but im unable to replicate it

**Comment by 0mlml** [comment]
Yeah, none of those are relevant to the script. What windows are you expecting to see? I reworked the UI and it should look like this:
![image](https://github.com/user-attachments/assets/163b9797-8cea-4bf4-9abb-a6ad1dc5c52b)

**Comment by Sciccorz** [comment]
Alr I got it to work on windows. On Mac none of the windows you showed are appearing and the screenshot was from Mac.

**Comment by HaiTechie** [comment]
the windows is not showing

**Comment by HaiTechie** [comment]
![image](https://github.com/user-attachments/assets/5b4fd761-71e3-41ef-9a0f-ab3780a514e9)

**Comment by HaiTechie** [comment]
![image](https://github.com/user-attachments/assets/30095532-f78a-44b5-ba07-c7c761169b42)
not with alt L also

**Comment by HaiTechie** [comment]
> Alr I got it to work on windows. On Mac none of the windows you showed are appearing and the screenshot was from Mac.

how

**Comment by stukov3** [comment]
сначала находясь на сайте chess.com прожми ALT+C у тебя должно открыться окно маленькое бесполезное(вроде) потом только нажми ALT+K я так починил всё хз правда как это работает

**Comment by Groxzzl** [comment]
Can this script be made to run as a third party? What I mean is, can it run outside of Chrome? Because if it runs in Chrome, it gets detected when I share my screen
